services/web-app/package.json

```diff
@@ -17,23 +17,28 @@
     "lint": "next lint",
     "service:build": "npm run build",
     "start": "next start",
-    "test": "echo 'No tests defined yet!'"
+    "test": "echo 'No tests defined yet!'",
+    "start-secure": "node server.js"
   },
   "bugs": {
     "url": "https://github.com/carbon-design-system/carbon-platform/issues"
   },
   "dependencies": {
+    "@carbon-platform/api": "0.1.3",
```

Comment:
With the changes for docker stuff, we no longer have to do any installs for local deps, so this can be removed via `npm -w services/web-app uninstall @carbon-platform/api`

Reply:
Done!